dubbo-common/pom.xml

```diff
@@ -100,8 +100,38 @@
             <groupId>javax.annotation</groupId>
             <artifactId>javax.annotation-api</artifactId>
         </dependency>
-
-
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>get-version-infos</id>
+                        <phase>compile</phase>
+                        <configuration>
+                            <failOnError>false</failOnError>
```

Comment:
Should enable `failOnError` in `release` profile

Reply:
Ok